stackdriver/config/config_test.go

```diff
@@ -0,0 +1,82 @@
+package config
+
+import (
+	corev1 "k8s.io/api/core/v1"
+
+	"testing"
+)
+
+func TestNewStackdriverConfigFromConfigMap(t *testing.T) {
+	tests := []struct {
+		name           string
+		configMap      *corev1.ConfigMap
+		expectedConfig Config
+	}{
+		{
+			name: "fullSdConfig",
+			configMap: &corev1.ConfigMap{
+				Data: map[string]string{
+					"project-id":           "project",
+					"gcp-location":         "us-west1",
+					"cluster-name":         "cluster",
+					"gcp-secret-name":      "secret",
+					"gcp-secret-namespace": "non-default",
+				},
+			},
+			expectedConfig: Config{
+				ProjectID:          "project",
+				GCPLocation:        "us-west1",
+				ClusterName:        "cluster",
+				GCPSecretName:      "secret",
+				GCPSecretNamespace: "non-default",
+			},
+		},
+		{
```

Comment:
nit: coalesce to 
```
}, {
```
see for example https://github.com/knative/serving/blob/master/pkg/reconciler/route/resources/names/names_test.go#L40